coins.json

```diff
@@ -87,13 +87,13 @@
         "symbol": "DGB",
         "decimals": 8,
         "blockchain": "Bitcoin",
-        "derivationPath": "m/44'/20'/0'/0'",
+        "derivationPath": "m/84'/20'/0'/0/0",
         "curve": "secp256k1",
         "publicKeyType": "secp256k1",
         "publicKeyHasher": "sha256ripemd",
         "base58Hasher": "sha256d",
-        "xpub": "zpub",
-        "xprv": "zprv",
+        "xpub": "xpub",
```

Comment:
bip44 -> xpub
bip84 -> zpub

Reply:
Set the followings values as per advice from DigiByte community, address verified against iancoleman.io

```
        "derivationPath": "m/84'/20'/0'/0/0",
        "xpub": "xpub",
        "xprv": "xprv",
```

Reply:
But it doesn’t make sense, if you use native segwit address, you should use zpub

Reply:
I think you are right that `BIP84` uses `zpub` \ `zprv`.
For some reason I got told to use `xpub` \ `xprv` by the DigiByte community, and https://iancoleman.io/bip39/ also uses `xpub` \ `xprv` for DGB segwit addresses, that is where I got source of truth for our tests.

Reply:
Done in 1f5f21b6d8896693b9e8bbcb4354012f19e9799e